event-exporter/example/event-exporter.yaml

```diff
@@ -44,7 +43,7 @@ spec:
     spec:
       serviceAccountName: event-exporter-sa
       containers:
-      - name: event-exporter
-        image: gcr.io/google-containers/event-exporter:v0.1.4
-        command:
-        - '/event-exporter'
+        - name: event-exporter
```

Comment:
one tab less?

Reply:
Ok let me do that. I think my yaml linter automatically indented this.

Reply:
Yeah the linter is saying that it should be indented (and judging by eye it looks like it should be too).
Still want me to de-indent?

Reply:
It doesn't matter, either way is correct in case of lists

In this particular case it's just how the whole file was formatted, why change?

Reply:
I'll de-indent (standard yml editor complained and did it was the original reason).

Reply:
You can just change the number manually, without editing the rest of the file

Reply:
Sorry I was editing this in vim (which indented automatically on save).
Is it ok now, or something I should change?